src/commands/create/groups/scripting.ts

```diff
@@ -8,10 +8,18 @@ import type { GenericTemplate } from "../index.js";
 
 type Template = GenericTemplate & {
   framework: "Node.js";
-  ethereumFramework: "Ethers v5" | "Ethers v6" | "viem";
+  ethereumFramework: "Ethers v5" | "Ethers v6" | "viem" | "web3js";
 };
 
 export const templates: Template[] = [
+  {
+    name: "Web3js - Node.js",
+    value: "web3js",
```

Comment:
```suggestion
    value: "node_web3js",
```
just to follow the same pattern as other templates